docker/dev_wrong_port_warning.py

```diff
@@ -5,20 +5,22 @@
 provides a helpful message to anyone still trying to use port 8000
 """
 
-import BaseHTTPServer
 import sys
+from http.server import BaseHTTPRequestHandler, HTTPServer
 
-class Handler(BaseHTTPServer.BaseHTTPRequestHandler):
+
+class Handler(BaseHTTPRequestHandler):
     def do_GET(self):
         self.send_response(503)
         self.end_headers()
         self.wfile.write(
-            'Your development environment is trying to connect to the KoBoCAT '
-            'container on port 8000 instead of 8001. Please change this. See '
-            'https://github.com/kobotoolbox/kobo-docker/issues/301 '
-            'for more details.'
+            b'Your development environment is trying to connect to the KoBoCAT '
+            b'container on port 8000 instead of 8001. Please change this. See '
+            b'https://github.com/kobotoolbox/kobo-docker/issues/301 '
+            b'for more details.'
```

Comment:
meh. since Python [now uses](https://www.python.org/dev/peps/pep-3120/) UTF-8 as the default source encoding, `b'blah blah'` should always produce an array of bytes containing a UTF-8-encoded string